tests/platforms/macOS/app/test_run.py

```diff
@@ -157,7 +162,9 @@ def test_run_gui_app_failed(run_command, first_app_config, sleep_zero, tmp_path)
     )
 
     with pytest.raises(BriefcaseCommandError):
-        run_command.run_app(first_app_config, passthrough=[])
+        run_command.run_app(
+            first_app_config, debugger_host=None, debugger_port=None, passthrough=[]
```

Comment:
Style nit - if the argument list is long enough to split like this, we prefer to go straight to "1 argument per line" form, rather than this form.